indra/newview/llgltfmateriallist.cpp

```diff
@@ -371,24 +371,19 @@ void LLGLTFMaterialList::queueApply(const LLViewerObject* obj, S32 side, const L
 {
     if (asset_id.isNull() || material_override == nullptr)
     {
-        queueApply(obj, side, asset_id);
-    }
-    else
-    {
-        LLGLTFMaterial* material = new LLGLTFMaterial(*material_override);
-        sApplyQueue.push_back({ obj->getID(), side, asset_id, material });
+        return;
     }
 
-    if (sUpdates.size() >= MAX_TASK_UPDATES)
+    // If face already has this asset id, avoid re-applying the same id; just modify override
+    if (obj && obj->getRenderMaterialID(side) == asset_id)
     {
-        LLCoros::instance().launch("modifyMaterialCoro",
-            std::bind(&LLGLTFMaterialList::modifyMaterialCoro,
-                gAgent.getRegionCapability("ModifyMaterialParams"),
-                sUpdates,
-                std::shared_ptr<CallbackHolder>(nullptr)));
-
-        sUpdates = LLSD::emptyArray();
+        queueModify(obj, side, material_override);
+        return;
```

Comment:
Our local id being accurate is an assumption not a fact. Generaly we assume that viewer might be not up to date and sensing an id is a good thing, but since that's a workaound for a server issue, please mark it as such.

It also adds a pitfall for future maintainers, because people will try to set render id first then will send it to server and it will fail.

Exmple:
`// Server side has a bug that makes server ignore override if asset id did not change, untill that is fixed, avoid re-applying the same id; just modify override`
LL_WARNS() << "No apparent change, Ignoring asset id and sending overrides only" << LL_ENDL;

P.S. I will create a server ticket to fix it.